styles.css

```diff
@@ -5,3 +5,27 @@
   margin: 0;
   padding: 0;
 }
+
+:root {
+  --logo-background: #0a0a23;
+}
+
+nav {
+  position: absolute;
+  width: 100%;
+  height: 80px;
+  background: var(--logo-background);
+  top: 0;
+  left: 0;
+  display: grid;
+  place-items: center;
+}
+
+img {
+  width: 336px;
+  height: 40px;
+}
+
+nav a {
+  cursor: pointer;
+}
```

Comment:
Links by default already have a cursor pointer so you don't need to add this here.